packages/node_modules/@node-red/nodes/locales/pt-BR/network/21-httprequest.html

```diff
@@ -0,0 +1,90 @@
+        <dd>Caso ocorram redirecionamentos durante o processamento da solicitação, esta propriedade é a URL redirecionada final.
+            Caso contrário, o url da solicitação original.</dd>
+        <dt>responseCookies<span class="property-type">objeto</span></dt>
+        <dd>Se a resposta incluir cookies, esta propriedade é um objeto de pares nome/valor para cada cookie.</dd>
+        <dt>redirectList<span class="property-type">matriz</span></dt>
+        <dd>Se a solicitação foi redirecionada uma ou mais vezes, as informações acumuladas serão adicionadas a esta propriedade. `location` é o próximo destino de redirecionamento. `cookies` são os cookies retornados da fonte de redirecionamento.</dd>
+    </dl>
+    <h3>Detalhes</h3>
+    <p>Quando configurada no nó, a propriedade URL pode conter tags <a href="http://mustache.github.io/mustache.5.html" target="_blank"> estilo bigode</a>. Estes permitem a
+    url a ser construída utilizando valores da mensagem de entrada. Por exemplo, se o url estiver definido para
+    <code>example.com/{{{topic}}}</code>, terá o valor de <code>msg.topic</code> inserido automaticamente.
+    Usar {{{...}}} evita que o bigode use caracteres de escape como /& etc.</p>
+    <p>O nó pode, opcionalmente, codificar automaticamente <code>msg.payload</code> como parâmetros de cadeia de caracteres de consulta para uma solicitação GET, caso em que <code>msg.payload</code> deve ser um objeto.</p>
+    <p><b>Nota:</b> Se estiver sendo executado atrás de um proxy, a variável de ambiente <code>http_proxy=...</code> padrão deve ser definida e o Node-RED reiniciado ou use então a configuração de proxy. Se a configuração do proxy foi definida, a configuração tem precedência sobre a variável de ambiente.</p>
+    <h4>Usando vários nós de solicitação HTTP</h4>
+    <p>Para usar mais de um desses nós no mesmo fluxo, deve-se ter cuidado com
+    a propriedade <code>msg.headers</code>. O primeiro nó irá definir esta propriedade com
+    os cabeçalhos de resposta. O próximo nó usará esses cabeçalhos para sua solicitação - isso 
+    geralmente não é a coisa certa a se fazer. Se a propriedade <code>msg.headers</code> for deixada inalterada
+    entre os nós, ela será ignorada pelo segundo nó. Para definir cabeçalhos personalizados, <code>msg.headers</code>
+    deve primeiro ser excluído ou redefinido para um objeto vazio: <code>{}</code>.
+    <h4>Tratamento de cookies</h4>
+    <p>A propriedade <code>cookies</code> passada ao nó deve ser um objeto de pares nome/valor.
+    O valor pode ser uma cadeia de caracteres para definir o valor do cookie ou pode ser um
+    objeto com uma única propriedade <code>value</code>.<p>
+    <p>Todos os cookies retornados pela solicitação são devolvidos à propriedade <code>responseCookies</code>.</p>
+    <h4>Tratamento do tipo de conteúdo</h4>
+    <p>Se <code>msg.payload</code> for um objeto, o nó definirá automaticamente o tipo de conteúdo
```

Comment:
It looks like a few paragraphs are missing here

Reply:
Thanks Again, Mauricio. I double-checked everything, file by file, and found some other small errors that I missed but they are all fixed now.